charts/k8s-job/README.md

```diff
@@ -0,0 +1,1174 @@
+# Kubernetes Service Helm Chart
+
+This Helm Chart can be used to deploy your job container under a
+[Job](https://kubernetes.io/docs/concepts/workloads/controllers/job/) resource onto your Kubernetes
+cluster. You can use this Helm Chart to run and deploy a one time job or periodic task such as a security scanner application or data science pipeline job.
+
+
+## How to use this chart?
+
+* See the [root README](/README.adoc) for general instructions on using Gruntwork Helm Charts.
+* See the [examples](/examples) folder for example usage.
+* See the provided [values.yaml](./values.yaml) file for the required and optional configuration values that you can set
+  on this chart.
+
+back to [root README](/README.adoc#core-concepts)
+
+## What resources does this Helm Chart deploy?
+
+The following resources will be deployed with this Helm Chart, depending on which configuration values you use:
+
+- `Job`: A standalone `Job` running the image specified in the
+                `containerImage` input value.
+
+back to [root README](/README.adoc#core-concepts)
+
+## How do I deploy additional services not managed by the chart?
+
+You can create custom Kubernetes resources, that are not directly managed by the chart, within the `customResources`
+key. You provide each resource manifest directly as a value under `customResources.resources` and set
+`customResources.enabled` to `true`. For examples of custom resources, take a look at the examples in
+[test/fixtures/custom_resources_values.yaml](../../test/fixtures/custom_resources_values.yaml) and
+[test/fixtures/multiple_custom_resources_values.yaml](../../test/fixtures/multiple_custom_resources_values.yaml).
+
+back to [root README](/README.adoc#day-to-day-operations)
+
+## How do I expose my application internally to the cluster?
```

Comment:
You don't need any of the "application" related sections for the Job README, but maybe something analogous that makes more sense. (I think you probably were aware of this, and just haven't gotten around to it yet.)